packages/@okta/vuepress-site/docs/guides/add-private-app/main/index.md

```diff
@@ -0,0 +1,99 @@
+
+This guide assumes that you've developed your SSO integration and want to add it to your Okta org. The instructions in this guide are generic for two SSO standards:
+
+* **OpenID Connect (OIDC)** (preferred)
+* **Security Assertion Markup Language (SAML)**
+
+> **Note:** Private integrations aren't restricted to the [OIN limitations](/docs/guides/submit-app-prereq/main/#oin-limitations). You can implement the Okta features that are available on your specific Okta org.
+
+## Create your private integration in Okta
+
+After you've built your SSO integration, you can use the Application Integration Wizard (AIW) in the Admin Console to create your app integration instance. This instance provides you with client credentials or metadata for your SSO flows.
+
+> **Note:** For best practices, create two or three additional admin users in Okta to manage the integration. This ensures that your team can access the integration for updates in the future.
+
+1. Sign in to your [developer-edition Okta org](/login/) as a user with administrative privileges.
+1. Go to **Applications** > **Applications** in the Admin Console.
+1. Click **Create App Integration**.
+
+<StackSnippet snippet="create" />
+
+<br>
+
+> **Note:** Creating your app integration instance doesn't automatically make it available in the [OIN](https://www.okta.com/integrations/). See [Publish an OIN integration](/docs/guides/submit-app-overview/) for an overview of the submission process.
+
+## Specify your integration settings
+
+After you create your integration instance, the main settings page appears for your new integration in the Admin Console. Specify **General Settings** and **Sign On** options, and assign the integration to users in your org. Click **Edit** if you need to change any of the options, and **Save** when you've made your changes.
+
+<StackSnippet snippet="settings" />
+
+## Test your integration
+
+This portion of the guide takes you through the steps required to test your integration.
+
+### Assign users
+
+First, you must assign your integration to one or more test users in your org:
+
+1. Click the **Assignments** tab.
+1. Click **Assign** and then select either **Assign to People** or **Assign to Groups**.
+1. Enter the appropriate people or groups that you want to have Single Sign-On into your application, and then click **Assign** for each.
+1. Verify the user-specific attributes for any people that you add, and then select **Save and Go Back**.
+1. Click **Done**.
+
+### Test Single Sign-On
+
+1. Sign out of your Okta org. Click **Sign out** in the upper-right corner of the Admin Console.
+1. Sign in to the Okta End-User Dashboard as the regular user that was assigned the integration.
+
+   > **Note:** If you sign in as a non-admin user to your Okta org from a browser, the End-User Dashboard appears. To access the End-User Dashboard from a mobile device, see [Okta End-User-Dashboard](https://help.okta.com/okta_help.htm?type=eu&id=ext_user_dashboard_overview).
```

Comment:
I think admins get dropped into the end-user dashboard on login too?

Reply:
Yes, you're correct. I've been using the `*-admin.okta.com` URL, so I forgot about that part. Thanks!